internal/statestorage/redis/redispb/shared.go

```diff
@@ -104,7 +103,12 @@ func MarshalToRedis(ctx context.Context, pb proto.Message, pool *redis.Pool) (er
 		//  something like parseTag() in src/encoding/json/tags.go
 		//field := strings.ToLower(pbInfo.Type().Field(i).Tag.Get("json"))
 		field := strings.ToLower(pbInfo.Type().Field(i).Name)
-		value := gjson.Get(jsonMsg, field)
+		value := ""
+		//value, err = strconv.Unquote(gjson.Get(jsonMsg, field).String())
+		value = gjson.Get(jsonMsg, field).String()
```

Comment:
I've noticed a lot of things stop working when the json field keys are of varying case. Is there a reason to restrict everything too lower?

Reply:
It's not ideal but probably okay to leave it this way until post 1.0 or someone is sufficiently motivated to take a stab at another approach